.palantir/revapi.yml

```diff
@@ -1,4 +1,14 @@
 acceptedBreaks:
+  "0.1033.0":
+    com.palantir.atlasdb:atlasdb-cassandra:
+    - code: "java.method.returnTypeChanged"
```

Comment:
Feels ok? I don't think we ever run the kvs implementations with a mis-aligned version?

Reply:
There's some internal usages:
* In internal DR product CoordinationServiceUtilities uses it.
I suspect this should be easey to fix.